src/SFA.DAS.ApplyService.Web/Views/Roatp/TaskList.cshtml

```diff
@@ -22,8 +21,8 @@
             <li>
                 <h2 class="app-task-list__section">
                     <span class="app-task-list__section-number">@Html.Raw(sequence.Id). </span>
-                    @Html.Raw(sequence.Description)
-                </h2>
+                    <a id="Sequence_@sequence.Id">@Html.Raw(sequence.Description)</a>
```

Comment:
Please revert this and just put an `id="Sequence_@sequence.Id"` as now all descriptions are hyperlinks (which they shouldn't be. Also they are not styled correctly ;)